backend/middlewares/validators/util.ts

```diff
@@ -101,6 +102,15 @@ export const validateContent = (content: ContentBlock): boolean => {
         flipCard.content.cards !== null && flipCard.content.cards.length >= 1
       );
     }
+    case "quiz": {
+      const quiz = content as QuizBlock;
+      return (
+        quiz.content.question !== null &&
+        ["MS", "MC"].includes(quiz.content.type) &&
```

Comment:
Nit: it seems a little unclear what these stand for. Maybe use longer strings to help with understanding?